sygnal/sygnal.py

```diff
@@ -186,9 +212,9 @@ async def make_pushkins_then_start(self):
         port = int(self.config["http"]["port"])
         for interface in self.config["http"]["bind_addresses"]:
             logger.info("Starting listening on %s port %d", interface, port)
-            self.reactor.listenTCP(port, pushgateway_api.site, interface=interface)
+            self.reactor.listenTCP(port, pushgateway_api.site, 50, interface=interface)
```

Comment:
Ahh yes, I remember this. PosixReactorBase provides a default value for this arg, even though IReactorTCP doesn't!

https://github.com/matrix-org/sydent/blob/7bdaac2774840e1f9697d30515f367c5c8c16866/sydent/http/httpserver.py#L148